integration/indexes_compat_test.go

```diff
@@ -559,7 +559,7 @@ func TestIndexesCompatDrop(t *testing.T) {
 	}
 }
 
-func TestIndexesCompatDropRunCommand(t *testing.T) {
+func TestIndexesDropCommandCompat(t *testing.T) {
```

Comment:
Okay so it's `TestDropIndexesCommandCompat` now according to the point 4.

Reply:
yes! thank you!